src/www/components/landingpage/LandingPage.scss

```
@@ -0,0 +1,612 @@
+.container {
+  background-image: linear-gradient(  rgb(129, 129, 129) ,white);
+  height: 325vh; //parallax(110vh) + 2 pages(200vh) + 15vh more empty space
```

You can just add a margin (top/bottom) instead of calculating manually how much space needs to be allocated.

```suggestion
  margin-bottom: 15vh;
```